guest_os.go

```diff
@@ -210,7 +210,7 @@ func update() {
 	}
 	d.OK()
 
-	exitStatus, runErr = run(`apt -o Dpkg::Options::="--force-confnew" full-upgrade --autoremove --purge --yes`)
+	exitStatus, runErr = run(`apt -o Dpkg::Options::="--force-confnew" full-upgrade --autoremove --purge --yes ` + strings.Join(flag.Args(), " "))
```

Comment:
I forgot what flag.Args() will get. Could you give me some examples?
```
ciel update-os arg1 arg2
=> ["arg1", "arg2"]
```
Am I correct?

Reply:
Also, how will the following cmdline be parsed?
```
ciel update-os -C "DIRECTORY" --dry-run --print-uris
```

Reply:
> Also, how will the following cmdline be parsed?
> 
> ```
> ciel update-os -C "DIRECTORY" --dry-run --print-uris
> ```

All parameters passed to `apt*` should be appended after `--`. Example in PR updated.